package.json

```diff
@@ -133,11 +133,6 @@
       "/node_modules/",
       "/dist/"
     ],
-    "clearMocks": true,
-    "globals": {
-      "ts-jest": {
-        "skipBabel": true
```

Comment:
Not removing this caused the coverage reports to not be using the original `.ts` file locations, but rather the transpiled `.js`!